backend/rates/history.go

```diff
@@ -49,16 +49,43 @@ var (
 	}
 )
 
+// EnableHistoryPair spins up a new goroutine to periodically update
+// and backfill historical exchange rates.
+// If a pair is unsupported or already enabled, EnableHistoryPair does nothing.
+// To stop periodic updates for this coin/fiat pair, the callers can use DisableHistoryPair.
+func (updater *RateUpdater) EnableHistoryPair(coin, fiat string) {
+	if geckoCoin[coin] == "" || geckoFiat[fiat] == "" {
+		return
+	}
+	updater.historyMu.Lock()
+	defer updater.historyMu.Unlock()
+	ctx, cancel := context.WithCancel(context.Background())
+	go updater.historyUpdateLoop(ctx, coin, fiat)
+	go updater.backfillHistory(ctx, coin, fiat)
+	updater.historyGo[coin+fiat] = cancel
```

Comment:
There's a mistake here. This shouldn't happen if already exists. I even wrote so in the doc comment. Will fix

Reply:
Do you mean this?

>  // If a pair is unsupported or already enabled, EnableHistoryPair does nothing.

As I don't see code that aborts early if already enabled.